src/one_dragon/base/operation/operation_notify.py

```
@@ -94,7 +108,10 @@ def send_application_notify(app: Application, status: bool | None) -> None:
         status_text = gt('开始')
 
     # 构建消息
-    _, app_name = _get_app_info(app)
+    try:
+        app_name = app.ctx.run_context.get_application_name(app_id)
+    except Exception:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)